js/web/lib/wasm/wasm-factory.ts

```diff
@@ -17,7 +17,7 @@ if (!BUILD_DEFS.DISABLE_TRAINING) {
       BUILD_DEFS.DISABLE_WEBGPU ? require('./binding/ort-wasm.js') : require('./binding/ort-wasm-simd.jsep.js');
 }
 
-const ortWasmFactoryThreaded: EmscriptenModuleFactory<OrtWasmModule> = !BUILD_DEFS.DISABLE_WASM_THREAD ?
+let ortWasmFactoryThreaded: EmscriptenModuleFactory<OrtWasmModule> = !BUILD_DEFS.DISABLE_WASM_THREAD ?
```

Comment:
@fs-eire, I was trying to move the code of `ortWasmFactory` and `ortWasmFactory` initialization to the `initializeWebAssembly` function, but failed.

Looks like it takes some other ort-web initialization work in somewhere. Is that possible to optimize them?

Reply:
The initialization logic is kind of complicated. You can check #18715 for details if you are interested.